docker-compose.child-env.yml

```diff
@@ -0,0 +1,98 @@
+services:
```

Comment:
currently i've only used this for spinning up integration environments in manual testing, not actually in any automated integration tests...but ideally we'd set up automated integration tests using a dual-app environment like this.